### ISSUE TITLE
Anyone uses YOLOv3 for pedestrian detection and got a sastified result???

### ISSUE THREAD

**Issue by Paul0629** [pjreddie/darknet#931]
I trained my yolo v3 using coco person and voc person. The loss is below 1. But the detect result is not that good while the pretrained model from the autuor works really well.  I don't know why. Anyone help?

**Comment by prnvjb** [comment]
@Paul0629  Could you send me the model?

**Comment by Paul0629** [comment]
My bad!!! I didn't set the batch and subdivision in the cfg file to be 1 when testing. It's much better now! I just don't know why we can't set batch and subdivision the same with training, like batch=64, subdivision=16.  @AlexeyAB @prnvjb

**Comment by basit26374** [comment]
@Paul0629 Can you please share you model ( .weghts and .cfg file) ?
I am also working on the same project. I have not enough resources like GPU's to re-trian the model.
